Data/Sys/GameSettings/R92.ini

```diff
@@ -0,0 +1,21 @@
+# R92E01, R92P01, R92J01 - Pikmin 2 Wii
```

Comment:
The non-alphabetical order is based on the order they're listed in the wiki for their associated pages, but I see now that it should be alphabetical like the rest of the inis. Thanks for scrutinizing things! I'll take care of this soon.